requirements.txt

```diff
@@ -5,4 +5,9 @@ pandas>=1.5,<2
 matplotlib>=3.6.0,<4
 numpy>=1.23.3,<2
 opensearch-py>=2
-protobuf==3.20.*
\ No newline at end of file
+protobuf==3.20.*
```

Comment:
can you please remove `protobuf==3.20.*` in your next commit? we don't need this anymore.

Reply:
sure

Reply:
resolved in 
[added save_as_pt function](https://github.com/opensearch-project/opensearch-py-ml/pull/18/commits/094339cca0a561d3ecd4edf74ce803b77e973239)